### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by surtic86** [JcPires/Project-Cars-DS-LiveApp#17]
How we want to handle the Translations? We go only with the Files? Or we go with a Crowd Translation Platform like crowdin.com or poeditor.com? As i know we can get a there on both a free Account because of the Project is OpenSource.

Only on File base its getting worse when we get more and more Translations. So this Tools would be really good to keep an Eye on the Translation Progress :)

What do you think?

**Comment by surtic86** [comment]
btw. i will start with the implementation of the placeholders for the translations.

**Comment by TylerBre** [comment]
I'm honestly not that familiar with i8n, we have a pretty robust system where I work that would be incredibly overkill for this app, but I can at least suggest that we support Portuguese, spanish, italian, french, and german, in that order. 

I hope we won't have to translate the logs, that would be a huge pain in the ass, haha.

**Comment by surtic86** [comment]
Yes that is so but when you support more then one Language its usable to use a Managing tool for it. We  will still use the Sails Translation stuff. But we mange the Srings over another Tool.

When you have more languages then 2 it will be really good. Also your Translators sees then directly what has changed and what she needs to translate.
